pom.xml

```diff
@@ -110,6 +110,12 @@
       <version>${cdap.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-etl-core</artifactId>
```

Comment:
why is this change needed?

Reply:
This dependency was left for by mistake, it should be removed.